core/db/migrate/20150826002555_convert_habtm_to_hmt_for_taxon_prototypes.rb

```diff
@@ -0,0 +1,15 @@
+class ConvertHabtmToHmtForTaxonPrototypes < ActiveRecord::Migration
+  def up
+    add_column :spree_taxons_prototypes, :created_at, :datetime
```

Comment:
you want `spree_taxons_promotion_rules` here right? also, you'll add a primary key as well yeah?